modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/views/table/TableHeadRow.js

```diff
@@ -35,6 +35,16 @@ const FieldsSelectorDropdown = ({fields}) => {
 	const [filteredFields, setFilteredFields] = useState(fields);
 	const [query, setQuery] = useState('');
 
+	const selectedFieldNames = Object.keys(visibleFieldNames).length
```

Comment:
Should we memo this? Not sure how costly this can become... probably not that much upfront, right?

Reply:
I thought about it, added `console.time()` to see how long it would take for just a few fields (3). It took 0.02ms. So even if I had like 50 fields (which is highly unlikely) it would still be insignificant so I figured I shouldn't other with `useMemo` for now.

Reply:
Sounds good, ship it then!

Reply:
- When to use `useMemo`: https://kentcdodds.com/blog/usememo-and-usecallback
- But also: https://github.com/liferay-frontend/liferay-portal/pull/287#discussion_r478486118 😀